pkg/volume/azure_dd/azure_common_linux.go

```diff
@@ -151,8 +151,9 @@ func formatIfNotFormatted(disk string, fstype string, exec mount.Exec) {
 		if err == nil {
 			// the disk has been formatted successfully try to mount it again.
 			glog.Infof("azureDisk - Disk successfully formatted (mkfs): %s - %s %s", fstype, disk, "tt")
+		} else {
+			glog.Warningf("azureDisk - format of disk %q failed: type:(%q) target:(%q) options:(%q)error:(%v)", disk, fstype, "tt", "o", err)
```

Comment:
What about `"error formatting volume with 'mkfs.%s %v': %v", fstype, args, err`? You're hiding "lazy_itable_init" and such.  (Applies also to `glog.Infof` few lines above).